ProcessorDefinitions/RecipientListDefinition.php

```diff
@@ -0,0 +1,94 @@
+<?php
+
+namespace Smartbox\Integration\CamelConfigBundle\ProcessorDefinitions;
+
+use Smartbox\Integration\FrameworkBundle\DependencyInjection\Traits\UsesEvaluator;
+use Symfony\Component\Config\Definition\Exception\InvalidConfigurationException;
+
+class RecipientListDefinition extends ProcessorDefinition
+{
+    use UsesEvaluator;
+
+    const AGGREGATION_STRATEGY_FIRE_AND_FORGET = 'fireAndForget';
+    const SIMPLE = 'simple';
+    const DELIMITER = ',';
+
+    /**
+     * {@inheritdoc}
+     */
+    public function buildProcessor($configNode, $id)
+    {
+        $definition = parent::buildProcessor($configNode, $id);
+
+        $evaluator = $this->getEvaluator();
+
+        $delimiter = (string) $configNode->attributes()->{'delimiter'};
```

Comment:
Maybe `$delimiter = (string) $configNode->attributes()->{'delimiter'}?: self::DELIMITER;` to make it simpler and remove the if below?

Reply:
Modified